apps/api/src/app/widgets/widgets.controller.ts

```diff
@@ -314,17 +314,3 @@ export class WidgetsController {
     return paramArray;
   }
 }
-
-/*
- * ValidationPipe convert boolean undefined params default (false)
- * Therefore we need to get string and convert it to boolean
- */
-export function initializeSeenParam(seen: string): boolean | null {
-  let isSeen: boolean = null;
-
-  if (seen) {
-    isSeen = seen == 'true';
-  }
-
-  return isSeen;
-}
```

Comment:
A small quesiton, @ShashwatAgrawal20 why did you removed the whole function? we need it back :)

Reply:
Oh okey, now seen your comment